default/python/AI/CombatRatingsAI.py

```diff
@@ -52,7 +53,7 @@ def __init__(self, attacks, structure, shields):
             """
 
             :param attacks:
-            :type attacks: dict|None
+            :type attacks: dict[float, int]|None
```

Comment:
Is it have sense to use float as a key of dict?

Reply:
Yes, the key is the damage of a weapon which is a float in the game engine.